test/parallel/test-vm-timeout.js

```diff
@@ -6,6 +6,9 @@ const vm = require('vm');
 // Test 1: Timeout of 100ms executing endless loop
 assert.throws(function() {
   vm.runInThisContext('while(true) {}', { timeout: 100 });
+}, function(err) {
```

Comment:
Haha I agree with you  before when I tried /^Script execution timed out\.$/ it didn't work, so I did the above approached. I didn't realize that "Error: " was included also. Awesome I will correct it!